artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/PostOfficeImpl.java

```diff
@@ -532,7 +532,7 @@ public AddressInfo updateAddressInfo(SimpleString addressName,
    public AddressInfo removeAddressInfo(SimpleString address) throws Exception {
       synchronized (addressLock) {
          server.callBrokerPlugins(server.hasBrokerPlugins() ? plugin -> plugin.beforeRemoveAddress(address) : null);
-         Bindings bindingsForAddress = getBindingsForAddress(address);
+         final Bindings bindingsForAddress = getDirectBindings(address);
```

Comment:
Looking at this, it seems it completely will ignore wildcard config. Im not so sure this is a good idea, as it takes into account these.

e.g. what happens if someone is using different wildcards? Also it be good to add test cases where alternative wildcards are used to make sure removeAddress still works.

Reply:
The getDirectBindings() method just does an equals() call and compares address names to see if the match exactly (ie simple strings).  It's not doing a match so there is no wild card involved.

Reply:
But I can still add some tests for a different wildcard anyways.